helm/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: deployment-operator
+description: creates a new instance of the plural deployment operator
+type: application
+version: 0.9.1
```

Comment:
Should we update to 0.1.0 or something like that?

Reply:
We'll probably have a lot of prereleases for it, we'll eventually go to a 0.x version when it's stable